mlir/include/mlir/Dialect/Arith/IR/ArithOps.td

```diff
@@ -1211,6 +1211,9 @@ def Arith_ExtFOp : Arith_FToFCastOp<"extf", [DeclareOpInterfaceMethods<ArithFast
     The destination type must to be strictly wider than the source type.
     When operating on vectors, casts elementwise.
   }];
+  let extraClassDeclaration = [{
+    bool isApplicable() { return true; }
```

Comment:
When is this method called?

Did you mean `isArithFastMathApplicable()` here? (if so we're missing a test to cover this)